typescript/cli/src/config/chain.ts

```diff
@@ -90,6 +90,62 @@ export async function createChainConfig({
     protocol: ProtocolType.Ethereum,
     rpcUrls: [{ http: rpcUrl }],
   };
+  const wantAdvancedConfig = await confirm({
+    message: 'Do you want to add advanced config for this chain?',
+  });
+  if (wantAdvancedConfig) {
+    const wantBlockConfig = await confirm({
+      message: 'Do you want to add block config for this chain?',
+    });
+    if (wantBlockConfig) {
+      const blockConfirmation = await input({
+        message:
+          'Enter no. of blocks to wait before considering a transaction confirmed(0-10)',
+        validate: (value) => parseInt(value) >= 0 && parseInt(value) <= 10,
+      });
+      const blockReorgPeriod = await input({
+        message:
+          'Enter no. of blocks before a transaction has a near-zero chance of reverting(0-50)',
+        validate: (value) => parseInt(value) >= 0 && parseInt(value) <= 50,
```

Comment:
Ditto, polygon uses 256 so how about 500 again

Reply:
Sure i will update it !!!